README.md

```diff
@@ -1,13 +1,13 @@
-WebExtensions / Chrome browser add-on for linking a browser to the KeePassRPC KeePass plugin.
+WebExtensions (Mozilla Firefox) / Google Chrome (Chromium) browser add-on for linking browser to KeePassRPC KeePass plugin.
```

Comment:
Please can we keep the "the" and go for a plural "browsers", otherwise I like the change.

"WebExtensions (Mozilla Firefox) / Google Chrome (Chromium) browser add-on for linking browsers to the KeePassRPC KeePass plugin."